core/src/main/java/org/stellar/anchor/MoreInfoUrlConstructor.java

```diff
@@ -0,0 +1,5 @@
+package org.stellar.anchor;
+
+public abstract class MoreInfoUrlConstructor {
```

Comment:
There is no implementation here. I would suggest to use interface.

Reply:
I don't think I get you comment. I have 2 separate `Sep6MoreInfoUrlConstructor` and `Se24MoreInfoUrlConstructor` extends it and implement the function. Can you elaborate more on the question?